requirements.txt

```diff
@@ -1,6 +1,6 @@
 apache-airflow>=2.3.0
```

Comment:
You should really be pinning exact versions (e.g. ==) in requirements file so we have repeatable tests.

Reply:
in most of the cases user can already have apache-airflow in place so we shouldn't limiting this

Reply:
This file should be only used for our internal testing so we should be explicit about what versions we pull down so our tests are repeatible. In `setup.py` (or pyproject.toml) we specify ranges of versions of packages we support, in this case we should try and be as broad as possible and have as open of ranges as possible to not create dependency conflicting scenarios for customers.

Reply:
I'm not saying we need to do it now but at some point in the future.